sdk/keyvault/azure-keyvault-certificates/migration_guide.md

```diff
@@ -0,0 +1,210 @@
+        certificate_name="cert-name",
+        certificate_version=certificate_version
+    )
+```
+
+Now in `azure-keyvault-certificates` you can retrieve the latest version of a certificate (as a `KeyVaultCertificate`) by using `get_certificate` and providing a certificate name.
+
+```python
+certificate = certificate_client.get_certificate(certificate_name="cert-name")
+
+print(certificate.name)
+print(certificate.properties.version)
+print(certificate.policy.issuer_name)
+```
+
+You can use `get_certificate_version` to retrieve a specific version of a certificate.
+
+```python
+certificate = certificate_client.get_certificate_version(certificate_name="cert-name", version="cert-version")
+```
+
+### List properties of certificates
+
+In `azure-keyvault` you could list the properties of certificates in a specified vault with the `get_certificates` method. This returned an iterator-like object containing `CertificateItem` instances.
+
+```python
+certificates = client.get_certificates(vault_base_url="https://my-key-vault.vault.azure.net/")
+
+for certificate in certificates:
+    print(certificate.attributes.created)
+    print(certificate.x509_thumbprint)
+```
+
+Now in `azure-keyvault-certificates` you can list the properties of certificates in a vault with the `list_properties_of_certificates` method. This returns an iterator-like object containing `CertificateProperties` instances.
+
+```python
+certificates = certificate_client.list_properties_of_certificates()
+
+for certificate in certificates:
+    print(certificate.name)
+    print(certificate.x509_thumbprint)
+```
+
+### Delete a certificate
+
+In `azure-keyvault` you could delete all versions of a certificate with the `delete_certificate` method. This returned information about the deleted certificate (as a `DeletedCertificateBundle`), but you you could not poll the deletion operation to know when it completed. This would be valuable information if you intended to permanently delete the deleted certificate with `purge_deleted_certificate`.
+
+```python
+deleted_certificate = client.delete_certificate(
+    vault_base_url="https://my-key-vault.vault.azure.net/",
+    certificate_name="cert-name"
+)
+
+# this purge would fail if deletion hadn't finished
+client.purge_deleted_certificate(
+    vault_url="https://my-key-vault.vault.azure.net/",
+    certificate_name="cert-name"
+)
+```
+
+Now in `azure-keyvault-certificates` you can delete a certificate with `begin_delete_certificate`, which returns a long operation poller object that can be used to wait/check on the operation much like you would with the poller received from `begin_create_certificate`. Calling `result()` on the poller will return information about the deleted certificate (as a `DeletedCertificate`) without waiting for the operation to complete, but calling `wait()` will wait for the deletion to complete. Again, `purge_deleted_certificate` will permanently delete your deleted certificate and make it unrecoverable.
```

Comment:
I might not mention `begin_create_certificate` here, to avoid muddying the waters. `result()` is blocking on that poller but not on the one returned by `begin_delete_certificate`.